src/main/java/org/spongepowered/common/mixin/core/entity/passive/MixinEntityOcelot.java

```diff
@@ -80,6 +83,14 @@ public int onTame(Random rand, int bound, EntityPlayer player, EnumHand hand) {
         return 1;
     }
 
+    @Inject(method = "processInteract", locals = LocalCapture.CAPTURE_FAILHARD,
+            at = @At(value = "FIELD", target = "Lnet/minecraft/entity/player/PlayerCapabilities;isCreativeMode:Z"))
+    public void onHealFeed(EntityPlayer player, EnumHand hand, CallbackInfoReturnable<Boolean> cir,
+            org.spongepowered.api.item.inventory.ItemStack itemStack) {
```

Comment:
wait, what method is this that it's using an API item stack? That doesn't make sense....